go/felix/intdataplane/endpoint_mgr.go

```diff
@@ -207,8 +207,24 @@ func (m *endpointManager) resolveWorkloadEndpoints() error {
 			var ipStrings []string
```

Comment:
Alternatively, this code might be simpler if we make a utility function that did the extraction from the proto endpoint and converted into a simpler struct holding just the information we need.